src/components/NavBar.jsx

```diff
@@ -0,0 +1,35 @@
+import { SlCompass } from "react-icons/sl";
+
+const NavBar = () => {
+
+    let Links=[
+        {name: "HOME", link:"/"},
+        {name: "UPCOMING TRIPS", link:"/upcoming-trips"},
+        {name: "TRAVEL HISTORY", link:"/history"},
+    ]
+
+    return (
+        <div className="shadow-md w-full fixed top-0 left-0">
+            <div className="md: flex items-center justify-between bg-transparent py-4 md:px-10 px-7">
+            <div className="font-bold text-2xl cursor-pointer flex items-center font-sans
+            text-gray-800">
+                <span className="text-2xl text-black mr-1 pt-2">
+                    <SlCompass name="compass-logo" size={24}/> 
+                </span>
+                Travelr
+            </div>
+            <ul className="md:flex md:items-center">
+               {
+                Links.map((link)=> (
+                    <li key={link.name} className="md:ml-8 text-l">
+                        <a href={link.link}>{link.name}</a>
+                    </li>
+                ))
+               }
+            </ul>
+            </div>
+        </div>
+    )
+}
+
+export default NavBar
```

Comment:
Should this have a semi colon?

Reply:
I don't think so - React doesn't add a semicolon in the default page when you first create the React app: 

![image](https://github.com/gbataglini/travel-roulette-ui/assets/118122049/10ec84cf-1ba3-4ab4-9677-3c4ac0362451)

I'll merge the PR